### ISSUE TITLE
Unmerged elements in stdlib

### ISSUE THREAD

**Issue by BarkingBad** [Kotlin/dokka#1485]
Some elements (mostly collections) in stdlib are multiplatform but, because no `expect`/`actual` relation are not merged into one. We should find a way to merge these elements without changing the default behaviour of not merging elements from different `sourceSets` with clashing `DRIs`

**Comment by BarkingBad** [comment]
List of elements with not merged elements
```
./kotlin-stdlib/kotlin.annotation/-must-be-documented(-common)
./kotlin-stdlib/kotlin.annotation/-target(-common)
./kotlin-stdlib/kotlin.annotation/-repeatable(-common)
./kotlin-stdlib/kotlin.annotation/-annotation-retention(-common)
./kotlin-stdlib/kotlin.annotation/-annotation-target(-common)
./kotlin-stdlib/kotlin.annotation/-retention(-common)
./kotlin-stdlib/kotlin.ranges/-int-range(-common)
./kotlin-stdlib/kotlin.ranges/-int-progression(-common)
./kotlin-stdlib/kotlin.ranges/-closed-range(-common)
./kotlin-stdlib/kotlin.ranges/-long-range(-common)
./kotlin-stdlib/kotlin.ranges/-char-progression(-common)
./kotlin-stdlib/kotlin.ranges/-char-range(-common)
./kotlin-stdlib/kotlin.ranges/-long-progression(-common)
./kotlin-stdlib/kotlin/-char-sequence(-common)
./kotlin-stdlib/kotlin/-parameter-name(-common)
./kotlin-stdlib/kotlin/-any(-common)
./kotlin-stdlib/kotlin/-char-array(-common)
./kotlin-stdlib/kotlin/-deprecation-level(-common)
./kotlin-stdlib/kotlin/-dsl-marker(-common)
./kotlin-stdlib/kotlin/-byte-array(-common)
./kotlin-stdlib/kotlin/-nothing(-common)
./kotlin-stdlib/kotlin/-deprecated(-common)
./kotlin-stdlib/kotlin/-byte(-common)
./kotlin-stdlib/kotlin/-double-array(-common)
./kotlin-stdlib/kotlin/-since-kotlin(-common)
./kotlin-stdlib/kotlin/-float(-common)
./kotlin-stdlib/kotlin/-char(-common)
./kotlin-stdlib/kotlin/-long(-common)
./kotlin-stdlib/kotlin/-boolean(-common)
./kotlin-stdlib/kotlin/-short-array(-common)
./kotlin-stdlib/kotlin/-int-array(-common)
./kotlin-stdlib/kotlin/-annotation(-common)
./kotlin-stdlib/kotlin/-short(-common)
./kotlin-stdlib/kotlin/-unsafe-variance(-common)
./kotlin-stdlib/kotlin/-long-array(-common)
./kotlin-stdlib/kotlin/-suppress(-common)
./kotlin-stdlib/kotlin/-float-array(-common)
./kotlin-stdlib/kotlin/-unit(-common)
./kotlin-stdlib/kotlin/-throwable(-common)
./kotlin-stdlib/kotlin/-number(-common)
./kotlin-stdlib/kotlin/-double(-common)
./kotlin-stdlib/kotlin/-comparable(-common)
./kotlin-stdlib/kotlin/-string(-common)
./kotlin-stdlib/kotlin/-array(-common)
./kotlin-stdlib/kotlin/-function(-common)
./kotlin-stdlib/kotlin/-int(-common)
./kotlin-stdlib/kotlin/-deprecated-since-kotlin(-common)
./kotlin-stdlib/kotlin/-boolean-array(-common)
./kotlin-stdlib/kotlin/-replace-with(-common)
./kotlin-stdlib/kotlin/-extension-function-type(-common)
./kotlin-stdlib/kotlin/-enum(-common)
./kotlin-stdlib/kotlin/-published-api(-common)
./kotlin-stdlib/kotlin.collections/-char-iterator(-common)
./kotlin-stdlib/kotlin.collections/-mutable-iterable(-common)
./kotlin-stdlib/kotlin.collections/-short-iterator(-common)
./kotlin-stdlib/kotlin.collections/-set(-common)
./kotlin-stdlib/kotlin.collections/-int-iterator(-common)
./kotlin-stdlib/kotlin.collections/-long-iterator(-common)
./kotlin-stdlib/kotlin.collections/-byte-iterator(-common)
./kotlin-stdlib/kotlin.collections/-mutable-list-iterator(-common)
./kotlin-stdlib/kotlin.collections/-iterable(-common)
./kotlin-stdlib/kotlin.collections/-map(-common)
./kotlin-stdlib/kotlin.collections/-mutable-iterator(-common)
./kotlin-stdlib/kotlin.collections/-float-iterator(-common)
./kotlin-stdlib/kotlin.collections/-mutable-set(-common)
./kotlin-stdlib/kotlin.collections/-mutable-map(-common)
./kotlin-stdlib/kotlin.collections/-list-iterator(-common)
./kotlin-stdlib/kotlin.collections/-collection(-common)
./kotlin-stdlib/kotlin.collections/-boolean-iterator(-common)
./kotlin-stdlib/kotlin.collections/-mutable-list(-common)
./kotlin-stdlib/kotlin.collections/-mutable-collection(-common)
./kotlin-stdlib/kotlin.collections/-iterator(-common)
./kotlin-stdlib/kotlin.collections/-double-iterator(-common)
./kotlin-stdlib/kotlin.collections/-list(-common)
```

**Comment by BarkingBad** [comment]
https://github.com/kamildoleglo/kotlin-dokka-stdlib/pull/3

**Comment by yalishevant** [comment]
fixed in 1.6.20 (#2316)
